JetAnalyzers/python/addAlgorithm.py

```diff
@@ -382,8 +388,26 @@ def addAlgorithm(process, alg_size_type_corr, Defaults, reco, doProducer):
         sequence = cms.Sequence(recJets * sequence)
         if type == 'PUPPI':
             process.load('CommonTools.PileupAlgos.Puppi_cff')
+            process.load('JetMETAnalysis.JetAnalyzers.customizePuppiTune_cff')
             #puppi.candName = cms.InputTag("particleFlow")
+            #80x change
+            puppiCentral[0].applyLowPUCorr = cms.bool(False)
+            puppiForward[0].applyLowPUCorr = cms.bool(False)
+            puppi.vertexName = "offlineSlimmedPrimaryVertices"
+            
+            #UpdatePuppiTuneV14(process)
+
+
+            
+            #puppi.UseFromPVLooseTight = cms.bool(False)
+            #puppi.UseDeltaZCut = cms.bool(False)
```

Comment:
Remove unused code. A history will be maintained in Git so nothing will be lost forever.